src/Core/Upload/ResumableUploader.php

```diff
@@ -40,6 +47,35 @@ class ResumableUploader extends AbstractUploader
      * @var string
      */
     private $resumeUri;
+    
+    /**
+     * Extend the parent constructor with the specific 
+     * for resumable upload option "uploadProgressCallback"
+     * 
+     * @param RequestWrapper $requestWrapper
+     * @param string|resource|StreamInterface $data
+     * @param string $uri
+     * @param array $options [optional] {
+     *     Optional configuration.
+     *
+     *     @type array $metadata Metadata on the resource.
+     *     @type callable $uploadProgressCallback to be called on each
+     *           successfully uploaded chunk.
+     *     @type int $chunkSize Size of the chunks to send incrementally during
+     *           a resumable upload. Must be in multiples of 262144 bytes.
+     *     @type array $httpOptions HTTP client specific configuration options.
+     *     @type int $retries Number of retries for a failed request.
+     *           **Defaults to** `3`.
+     *     @type string $contentType Content type of the resource.
+     * }
+     */
+    public function __construct(RequestWrapper $requestWrapper, $data, $uri, array $options = array())
```

Comment:
We've been using the shorthand array syntax (`[]`) through the project, would you mind updating so it is consistent?